.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,7 +16,9 @@ Rules:
 To avoid multiple iterations of fixes and CIs failures, please read
 https://checkstyle.org/contributing.html
 
-ATTENTION: We are not merging Pull Requests that are not passing our CIs,
-but we will help to resolve issues.
+ATTENTION: We are not merging Pull Requests that are not passing our CIs. You
+can't run Travis CI on your GitHub fork because it's limited to master and we
```

Comment:
Travis is doing only small part of validation. We consider it as primary due to historical reasons.
There bunch of others with the same policy - merge only on passed CI.
Contributor can run Travis in their fork, they just need to place fix in forked master ;).